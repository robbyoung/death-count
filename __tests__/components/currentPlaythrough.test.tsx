import React from 'react';
import renderer from 'react-test-renderer';
import CurrentPlaythrough from '../../app/components/currentPlaythrough';

describe('Current Playthrough Component', () => {
    it('renders correctly', () => {
        const component = renderer.create(<CurrentPlaythrough
            gameName="Test Game"
            playthroughName="Playthrough 1"
            onPress={() => undefined}
        />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});