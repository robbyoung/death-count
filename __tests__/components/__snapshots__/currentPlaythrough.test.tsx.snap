// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current Playthrough Component renders correctly 1`] = `
<View
  accessible={true}
  clickable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "marginBottom": 30,
      "opacity": 1,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 30,
        "textAlign": "center",
        "textAlignVertical": "center",
      }
    }
  >
    Playthrough 1
  </Text>
  <Text
    style={
      Object {
        "color": "#a9a9a9",
        "fontSize": 24,
        "textAlign": "center",
        "textAlignVertical": "center",
      }
    }
  >
    Test Game
  </Text>
</View>
`;
